packages/blocks/src/render-blocks.js

```diff
@@ -62,14 +62,34 @@ const parseNodes = ( nodes ) =>
 			name: toLower( node.tagName ),
 			children: parseNodes( node.childNodes ),
 			props: {
-				...attributes,
+				...parseTagProps( toLower( node.tagName ), attributes ),
 				className,
 				style: parseStyles( style ),
 				key: uniqueId( KEY_PREFIX ),
 			},
 		};
 	} );
 
+/**
+ * Parses DOM nodes into an array of objects to be used to create React elements.
+ *
+ * @param  {string} tagName
+ * @param {Object}  props object.
+ * @return {Object}  Props object.
+ */
+const parseTagProps = ( tagName, props ) => {
+	switch ( tagName ) {
+		case 'audio':
+			if (
+				typeof props.controls !== 'undefined' &&
+				props.controls !== false
+			) {
+				props.controls = 'controls';
+			}
```

Comment:
`break;` statement is missing.  

Personally, I'd prefer to get rid of the `switch` altogether and replace it with simple `if` statements and early returns.

Reply:
I changed this into a simple if, yet with its current use there's no point for an early return. Eventually other exceptions might need to be added but, for now, mutating the props and return at the end should suffice. Remember this function needs to return unmodified props when no tagName is matched